src/main/resources/db/migration/V1.0.3__init_tables.sql

```diff
@@ -0,0 +1,72 @@
+DROP TABLE IF EXISTS execution_types;
+CREATE TABLE execution_types
+(
+    execution_code execution_type_enum,
+    execution_name TEXT,
+    description    TEXT,
+    PRIMARY KEY (code)
+);
+
+DROP TABLE IF EXISTS tasks;
+CREATE TABLE tasks
+(
+    task_id                 TEXT,
+    task_name               TEXT,
+    task_type               TEXT,
+    due_date_time           TIMESTAMP,
+    state                   task_state_enum,
+    task_system             task_system_enum,
+    security_classification security_classification_enum,
+    title                   TEXT,
+    description             TEXT,
+    notes                   JSONB,
+    major_priority          INTEGER,
+    minor_priority          INTEGER,
+    assignee                TEXT,
+    auto_assigned           BOOLEAN   default false,
+    execution_type_code     execution_type_enum,
+    work_type               TEXT,
+    role_category           TEXT,
+    has_warnings            BOOLEAN   default false,
+    assignment_expiry       TIMESTAMP,
+    case_id                 TEXT,
+    case_type_id            TEXT,
+    case_category           TEXT,
+    case_name               TEXT,
+    jurisdiction            TEXT,
+    region                  TEXT,
+    region_name             TEXT,
+    location                TEXT,
+    location_name           TEXT,
+    business_context        business_context_enum,
+    termination_reason      TEXT,
+    created                 TIMESTAMP default CURRENT_TIMESTAMP,
+    PRIMARY KEY (task_id),
+    CONSTRAINT fk_execution_type_code
+        FOREIGN KEY (execution_type_code)
+            REFERENCES execution_types (code)
```

Comment:
FOREIGN KEY (execution_code)?

Reply:
that one is okay the reference however should change

Reply:
```suggestion
            REFERENCES execution_types (execution_code)
```